life_server/Functions/Systems/fn_cleanup.sqf

```diff
@@ -73,8 +83,8 @@ for "_i" from 0 to 1 step 0 do {
         _x params ["_params", "_function", "_delay", "_timeToRun"];
         if (diag_tickTime > _timeToRun) then {
             _params call _function;
-            _x set [2, diag_tickTime + _delay];
+            _x set [3, diag_tickTime + _delay];
         };
     } forEach _routines;
-    uiSleep 1e-6;
+    uiSleep 60;

```

Comment:
Not too sure about the changes to this file, what's the intention? It seems you want to run the procedures and cleanup every 60 seconds? Some of the procedures do not need to be called often and are usually only ran once on server start

Reply:
well - 1e-6 = 0.000001 = 0.001ms - so I just fixed several orders of magnitute